.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: ruby
 
+cache: bundler
+
 bundler_args: --without documentation
 
 rvm:
-  - 2.6.2
-  - 2.5.5
-  - 2.4.5
-  - 2.3.8
+  - 2.7
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
```

Comment:
Simplified config, reduces maintenance burden a bit and Travis slowly but surely uses the latest versions over time in my experience.